edx_proctoring/tests/test_views.py

```diff
@@ -901,77 +808,6 @@ def test_attempt_status_stickiness(self):
                 ProctoredExamStudentAttemptStatus.submitted
             )
 
-    @ddt.data(
-        ProctoredExamStudentAttemptStatus.created,
-        ProctoredExamStudentAttemptStatus.ready_to_start,
-        ProctoredExamStudentAttemptStatus.started,
-        ProctoredExamStudentAttemptStatus.ready_to_submit
-    )
-    def test_attempt_callback_timeout(self, running_status):
-        """
-        Ensures that the polling from the client will cause the
-        server to transition to timed_out if the user runs out of time
```

Comment:
Is this in addition to the banner polling?  Wouldn't this have already happened with the banner polling?  Can we create a 1 minute exam and test?